doc/example/example_Isensee/Isensee_measurementData.tsv

```diff
@@ -0,0 +1,688 @@
+observableId	preequilibrationConditionId	simulationConditionId	measurement	time	observableParameters	noiseParameters	observableTransformation	noiseDistribution	experimentId	datasetId	replicateId
+pRII_Microscopy	control	JI09_150302_Drg345_343_CycNuc__4_ABnOH_and_ctrl__1	986.688213044207	65	s_pRII_JI09_150302_Drg345_343_CycNuc;s_pRII_global;b_pRII_global;rel_open;xi_rel_open	s_pRII_JI09_150302_Drg345_343_CycNuc;s_pRII_global;rho_pRII_Microscopy	lin	normal	JI09_150302_Drg345_343_CycNuc	JI09_150302_Drg345_343_CycNuc__4_ABnOH_and_ctrl	Drg343
+pRII_Microscopy	control	JI09_150302_Drg345_343_CycNuc__4_ABnOH_and_ctrl__2	963.146739077231	65	s_pRII_JI09_150302_Drg345_343_CycNuc;s_pRII_global;b_pRII_global;rel_open;xi_rel_open	s_pRII_JI09_150302_Drg345_343_CycNuc;s_pRII_global;rho_pRII_Microscopy	lin	normal	JI09_150302_Drg345_343_CycNuc	JI09_150302_Drg345_343_CycNuc__4_ABnOH_and_ctrl	Drg343
```

Comment:
ok, does it really have to be such large files :)

Reply:
Isensee was chosen because it covers a few cases on its own

Reply:
As I pointed out:
We could think of using simpler, artificial examples. Computation time is the major drawback of using real example cases.
There are two major advantages, however:
1. Less time to implement the tests
2. Tests do indeed cover real situations and not just artificial stuff.

Reply:
Should be fine :+1: